R/join-by-compat.R

```diff
@@ -0,0 +1,96 @@
+new_join_by <- function(exprs, condition, filter, cross, x, y) {
+  out <- list(
+    exprs = exprs,
+    condition = condition,
+    filter = filter,
+    cross = cross,
```

Comment:
Note that we removed `cross` from `join_by()` altogether in favor of people using `cross_join()` instead

Reply:
Thanks for the info!
I strongly prefer `cross_join()` over the `join_by()` variant 😄